### ISSUE TITLE
ssh-pool:1.4.0 support

### ISSUE THREAD

**Issue by Zhomart** [shipitjs/shipit#140]
On version `ssh-pool:1.4.0`, they've added proxy support, which breaks shipit.js.

I think one of the following should've been done:

- `ssh-pool` should've added proxy support on version 2.x 
- shipit.js's package.json `"ssh-pool": "^1.0.0",` should be changed to `"ssh-pool": "~1.0.0",`.

**Comment by Zhomart** [comment]
I fixed putting following into `shipitfile.js`

```js
module.exports = function (shipit) {

  // Fix for the issue https://github.com/shipitjs/shipit/issues/140
  shipit.initSshPool = function () {
    if (!this.config.servers)
      throw new Error('Servers not filled');

    var servers = _.isArray(this.config.servers) ? this.config.servers : [this.config.servers];
    if (sshPool.ConnectionPool.length === 2){
      this.pool = new sshPool.ConnectionPool(servers, _.extend({}, this.options, _.pick(this.config, 'key', 'strict')));
    } else {
      this.pool = new sshPool.ConnectionPool(servers, null, _.extend({}, this.options, _.pick(this.config, 'key', 'strict')));
    }
    return this;
  };

  // ...
}
```

**Comment by timkelty** [comment]
@Zhomart you're right. For some reason I thought it wasn't a breaking change...I'll get this fixed tonight.

**Comment by timkelty** [comment]
@Zhomart I published ssh-pool@1.4.1 that reverts the proxy changes.

I'll review feature again and release it with a proper version.
Sorry about that!
